tests/e2e/pkg/helpers/ns/namespace.go

```diff
@@ -0,0 +1,51 @@
+package ns
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kyma-project/istio/operator/tests/e2e/e2e/setup"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/klient/k8s"
+	"sigs.k8s.io/e2e-framework/klient/k8s/resources"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+)
+
+type CreateOpts func(object k8s.Object)
+
+func WithLabels(labels map[string]string) CreateOpts {
+	return func(object k8s.Object) {
+		object.SetLabels(labels)
+	}
+}
+
+func CreateNamespace(ctx context.Context, t *testing.T, name string, cfg *envconf.Config, opts ...CreateOpts) error {
+	r, err := resources.New(cfg.Client().RESTConfig())
```

Comment:
Is there an option to use the k8s client with logging?
`tests/e2e/e2e/setup/client.go`

Reply:
Looks like so.
it uses the same rest.Config struct for instantiating client.